### DISCUSSION TITLE
Facing AttributeError: module 'openfl.interface.collaborator' has no attribute 'create' error while running Federated_FedProx_PyTorch_MNIST_Tutorial jupyter notebook.

### DISCUSSION THREAD

**Discussion by tanwarsh** [securefederatedai/openfederatedlearning#1061] [Q&A] [answered]
Hi,

While running jupyter notebook `AttributeError: module 'openfl.interface.collaborator' has no attribute 'create'` from a docker container I am getting error mentioned below:

```shell
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[9], line 1
----> 1 fx.init('torch_cnn_mnist')

File [~/.local/lib/python3.10/site-packages/openfl/native/native.py:242](http://10.65.47.207:8888/home/openfl/.local/lib/python3.10/site-packages/openfl/native/native.py#line=241), in init(workspace_template, log_level, log_file, agg_fqdn, col_names)
    240 data_path = 1
    241 for col_name in col_names:
--> 242     collaborator.create(col_name, str(data_path), silent=True)
    243     collaborator.generate_cert_request(col_name, silent=True, skip_package=True)
    244     collaborator.certify(col_name, silent=True)

AttributeError: module 'openfl.interface.collaborator' has no attribute 'create'
```

To Reproduce issue:
1. Pull code from develop branch
2.  `./scripts/build_base_docker_image.sh` to build docker image
3. `docker run -it --network host openfl`
4. `jupyter notebook --ip='*' --port=8080 --no-browser`
5. Then run `Federated_FedProx_PyTorch_MNIST_Tutorial`

How can I resolve this issue?

**Comment by MasterSkepticista** [comment, marked as answer]
Related: https://github.com/securefederatedai/openfl/issues/834

It seems `create(...)` was removed in this commit: https://github.com/securefederatedai/openfl/pull/1003/commits/4abfb0a12c183ec123f9775613fa56b36eb69b24

Edit: @tanwarsh Can you test your code with #1062?
